updates/version.yaml

```diff
@@ -48,3 +48,4 @@
     - Added metadata column for plugins to store data in
     - posts_add_metadata.php
 1.3.1: Fixed metadata column not being jsonable
+1.3.2: Fix override of default slug param names with custom ones
```

Comment:
@GinoPane If you don't mind, would you be able to remove this change? We will update the `version.yaml` file when we publish the next version. :)